netkan/netkan/status.py

```diff
@@ -0,0 +1,103 @@
+import boto3
+import json
+import logging
+import os
+import time
+from datetime import datetime
+from dateutil.parser import parse
+from pathlib import Path
+from pynamodb.models import Model
+from pynamodb.attributes import (
+    UnicodeAttribute, UTCDateTimeAttribute, BooleanAttribute
+)
+
+
+# The click context isn't available during the setup of this object and the
+# idea is that you'd specify a different table using a different class. Since
+# this is for dev and dev won't have access to the production table taking a
+# environment variable to override is acceptable for now.
+def table_name():
+    return os.getenv('STATUS_DB', 'NetKANStatus')
+
```

Comment:
This is what I was referring to @HebaruSan

Reply:
Yup, I saw that. It was what prompted me to look more closely at the `--status_db` params, in fact.